crates/parser/src/zombienet.pest

```diff
@@ -83,6 +85,10 @@ pause = { node_name ~ "pause" }
 resume = { node_name ~ "resume" }
 restart = { node_name ~ "restart" ~ ("after" ~ int+ ~ seconds)? }
 
+// CHAOS COMMANDS
+setter = { node_name ~ "set" ~ chaos_type ~ "by" ~ int+ ~ (seconds | miliseconds)}
+remover = { node_name ~ "remove" ~ chaos_type }
```

Comment:
```suggestion
chaos_setter = { node_name ~ "set" ~ chaos_type ~ "by" ~ int+ ~ (seconds | miliseconds)}
chaos_remover = { node_name ~ "remove" ~ chaos_type }
```

Reply:
This will also only allow to set one of the two options (latency/jitter), I think we can work to cover all 3 options in the same command (allow to set latency / jitter / latency and jitter).